docs/install.md

```diff
@@ -1,5 +1,4 @@
 ---
-title: How to install DietPi
```

Comment:
Here we should leave it, since "Installation" is too unspecific.

Reply:
I thought it would be sufficient if the following H1 headline below is "How to install DietPi".
But now I understand that the (left) navigation bar title is relevant. Right?

Reply:
> But now I understand that the (left) navigation bar title is relevant. Right?

Exactly.

Reply:
Ok, then I have to review the changes.